src/Symfony/Component/Config/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": "^7.1.3",
+        "ext-dom": "*",
```

Comment:
although DOM is installed together with `php-xml`, it's still possible to disable `xml extension` and enable `dom extension`

```
// test.php
<?php
use Symfony\Component\Config\Util\XmlUtils;
require_once 'vendor/autoload.php';
var_dump(XmlUtils::loadFile('test.xml'));
```

```
// test.xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns="http://example.com/schema"></root>
```

```
$ php -n -dextension=phar.so -dextension=json.so -dextension=iconv.so -dextension=xml.so test.php
Fatal error: Uncaught Error: Class 'DOMDocument' not found in /mnt/c/checkouts/symfony/src/Symfony/Component/Config/Util/XmlUtils.php:52
$ echo $? # => 255
```

```
$ php -n -dextension=phar.so -dextension=json.so -dextension=iconv.so -dextension=dom.so test.php
$ echo $? // => 0
```